xmipp

```diff
@@ -449,7 +449,8 @@ def printVersion():
     with open(VERSION_FILE, 'w') as f:
         f.write(str(log))
```

Comment:
- Is log a list?
- What should str(log) return?
- How does f.write(list) work? Does it convert each element into a string and writes it to a file, or does it work as a f.write(str(list))?
- Should log2 be checked as non-empty before f.write(log2[0])?

Reply:
Changes so far look fine, those are only questions that might lead to suggestions to further improve the code

Reply:
log is a list if Build is nos found, else became a string
str(log) fixed
f.write() just writes if is a str
log3 must be checked yes